src/Workleap.DomainEventPropagation.Tests/Subscription/RegistrationTest.cs

```diff
@@ -15,11 +16,22 @@ public class RegistrationTest
     public async Task GivenDomainEventIsFired_WhenDomainEventHandlerIsRegisteredToMultipleDomainEvents_ThenDomainEventHandlerIsCalled()
     {
         var services = new ServiceCollection();
+
+        var configuration = new ConfigurationBuilder()
+            .AddInMemoryCollection(new Dictionary<string, string>
+            {
+                { $"{EventPropagationSubscriberOptions.SectionName}:SubscribedTopics:0", OrganizationTopicName },
+            })
+            .Build();
+
+        services.AddSingleton<IConfiguration>(configuration);
```

Comment:
Here, I choose to bind the topic from the new default configuration section instead of using the overload where topics are configured. Essentially, it is very similar to what was on the line 19 in the previous version of the file.